src/js/Controllers/UIController.js

```diff
@@ -401,7 +467,13 @@ class UIController {
     onChoiceSelection(choiceID, appID, msgID) {
         clearTimeout(this.timers[msgID])
         delete this.timers[msgID]
-        this.listener.send(RpcFactory.UIPerformInteractionResponse(choiceID, appID, msgID))
+
+        let imageValidationSuccess = GetValidationResult(msgID)
+        RemoveImageValidationRequest(msgID)
+
+        const rpc = (imageValidationSuccess) ? RpcFactory.UIPerformInteractionResponse(choiceID, appID, msgID)
+            : RpcFactory.InvalidImageResponse({ id: msgID, method: "UI.PerformInteraction" });
```

Comment:
I'm not sure about this one, if an image is failing to load perform interaction won't be able to get a choiceID back. I think the parameter (choiceID) should take precedent over the info (image validation) here. Unfortunately there is no way to have both params and info. What do you think?

Reply:
I think you're right. PerformInteraction should still function normally even if the images in the choice set are invalid. 

Since we can't return the choiceId in a warnings response I've removed the image validation for the perform interaction request in https://github.com/smartdevicelink/generic_hmi/pull/314/commits/c0000f7625983cdbecaaa77176254ec3513e0465